source/clear-linux/how-to/swupd/run.rst

```diff
@@ -1,59 +1,60 @@
 .. _swupd-run:
 
-How to run the updater
-######################
+How to update Clear Linux
+#########################
 
-This section describes how to manually trigger the software update code in
-Clear Linux\* OS for Intel® Architecture.
+This section describes how to update |CLOSIA|.
 
 Prerequisites
 =============
 
--  System is up and running a Clear Linux OS for Intel
-   Architecture official build, 7420 later.
+* The system is up and running the Clear Linux 15400 build or later.
 
--  Device is on a well-connected network.
+* Device is on a well-connected network.
 
--  Device is able to connect to release infrastructure. For example:
-   http://update.clearlinux.org/update/300/
+* Device is able to connect to the release infrastructure. For example:
+  http://update.clearlinux.org/update/300/
 
-Running the updater
-===================
+How to update the system
+========================
 
-#. Open a Terminal Emulator.
+Starting with version 15400, Clear Linux supports auto-update. By default, it
+is turned on.
 
-#. Become root and trigger an update with this command:
+To verify the current auto-update setting:
 
    .. code-block:: console
 
-      $ swupd update
+      $ swupd autoupdate
 
-#. If the updater console output indicates a kernel update occurred,
-   then at your convenience reboot to begin using the new OS version.
+To enable auto-update:
 
-Updater options
-===============
+   .. code-block:: console
+
+      $ sudo swupd autoupdate --enable
+
+To disable auto-update:
 
-Help options:
+   .. code-block:: console
+
+      $ sudo swupd autoupdate --disable
 
-.. code-block:: console
+If you wish to force a manual update:
 
-   -h, --help Display help options.
+   .. code-block:: console
 
-Application options:
+      $ sudo swupd update
 
-.. code-block:: console
+.. note::
+
+   If the updater console output indicates a kernel update occurred, then at
+   your convenience reboot to begin using the new OS version.
```

Comment:
Agree.  Let's drop "at your convenience".  So it should say something like... "If the swupd console output indicates a kernel update has occurred, then reboot immediately in order for the enhancements to take effect."